src/components/dev-hub/SEO.js

```diff
@@ -13,7 +13,15 @@ const getImageSrc = (src, siteUrl) => {
     return isExternalUrl.test(src) ? src : siteUrl + src;
 };
 
-const SEO = ({ articleTitle, ogTitle, image, url, type, twitterNode }) => {
+const SEO = ({
+    articleTitle,
+    canonicalUrl,
+    ogTitle,
+    image,
+    url,
```

Comment:
Super nit, but perhaps now is a good time to rename `url` to `ogUrl` to be extra specific, since it would only be used for the `og:url` tag